core/operation/src/main/java/uk/gov/gchq/gaffer/operation/OperationChain.java

```diff
@@ -97,23 +98,9 @@ void setOperationArray(final Operation[] operations) {
 
     @Override
     public String toString() {
-        final StringBuilder strBuilder = new StringBuilder("OperationChain[");
```

Comment:
I've checked with @p013570 and we decided to keep them uniform. The output of this is expressive enough.

`uk.gov.gchq.gaffer.operation.OperationChain@598067a5[
operations=[
uk.gov.gchq.gaffer.operation.impl.add.AddElements@3c0ecd4b, uk.gov.gchq.gaffer.operation.impl.get.GetAllElements@14bf9759
]]`